utils/pinprocfw/pinprocfw.cpp

```diff
@@ -32,12 +32,18 @@
 #endif  /* DEBUG_MODE */
 
 #include "pinprocfw.h"
-#include "../../src/PRHardware.h"
```

Comment:
Moving `const int` declarations to macros in `pinproc.h` allowed for removal of this header.